helicopyter.py

```diff
@@ -135,12 +138,18 @@ def multisynth(
 
 class Parameters(Tap):
     conas: list[str]  # space-separated COdeNAmes
+    directory: Path | None = None
     hashicorp_configuration_language: bool = False
 
     def configure(self) -> None:  # noqa: D102
-        self.add_argument('conas')  # Make this a positional argument
+        self.add_argument('conas')  # Positional argument
```

Comment:
This seems like it could be a good candidate for some kind of metaprogramming automation

Reply:
Yeah, https://pypi.org/project/typed-argument-parser/ falls back to argparse style in more situations than I'd like.